shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -4,6 +4,8 @@
 
 package io.flutter.view;
 
+import static io.flutter.plugin.platform.SingleViewPresentation.PresentationContext;
```

Comment:
this is just an inner class right? Why the static?

Reply:
`SingleViewPresentation.PresentationContext` makes the line longer. This is an inner/static class.

Reply:
You can import inner classes directly no? You're probably just running into an IDE/lint issue rather than a Java issue.

Reply:
you right. Maybe I thought this was a method.